core/frontend/src/main/scala/io/udash/bindings/inputs/RadioButtons.scala

```diff
@@ -0,0 +1,104 @@
+package io.udash.bindings.inputs
+
+import java.{util => ju}
+
+import io.udash._
+import io.udash.properties.PropertyCreator
+import org.scalajs.dom.{Element, Event, Node}
+import org.scalajs.dom.html.{Div, Input => JSInput}
+import scalatags.JsDom
+import scalatags.JsDom.all._
+
+/**
+  * Radio buttons group for finite options with one element selection.
+  */
+object RadioButtons {
+  def inputsOnlyDecorator[T]: Seq[(JSInput, T)] => Seq[Node] = _.map(_._1)
```

Comment:
Can we default to ISeq everywhere? These are all new methods.

Also: use pattern matching instead of `._1`

Reply:
Not really, if you meant these three decorators by "everywhere". `Seq` in `SeqProperty` is the root of this problem.